Sources/iOS/IndexPathConverter.swift

```diff
@@ -1,35 +1,39 @@
 import Foundation
 
-struct ChangeWithIndexPath {
-  let inserts: [IndexPath]
-  let deletes: [IndexPath]
-  let replaces: [IndexPath]
-  let moves: [(from: IndexPath, to: IndexPath)]
+public struct ChangeWithIndexPath {
+    public let inserts: [IndexPath]
+    public let deletes: [IndexPath]
+    public let replaces: [IndexPath]
+    public let moves: [(from: IndexPath, to: IndexPath)]
 }
 
-final class IndexPathConverter {
-  func convert<T>(changes: [Change<T>], section: Int) -> ChangeWithIndexPath {
-    let inserts = changes.flatMap({ $0.insert }).map({ $0.index.toIndexPath(section: section) })
-    let deletes = changes.flatMap({ $0.delete }).map({ $0.index.toIndexPath(section: section) })
-    let replaces = changes.flatMap({ $0.replace }).map({ $0.index.toIndexPath(section: section) })
-    let moves = changes.flatMap({ $0.move }).map({
-      (
-        from: $0.fromIndex.toIndexPath(section: section),
-        to: $0.toIndex.toIndexPath(section: section)
-      )
-    })
+public class IndexPathConverter {
+    public init() {
 
-    return ChangeWithIndexPath(
-      inserts: inserts,
-      deletes: deletes,
-      replaces: replaces,
-      moves: moves
-    )
-  }
+    }
+    
+    public func convert<T>(changes: [Change<T>], section: Int) -> ChangeWithIndexPath {
```

Comment:
I have convention of `2 spaces indentation`, can you use that?

Reply:
Sure, sorry for my sloppiness. I will fix it.